### PR TITLE
Minor readme fixes

### PR DESCRIPTION
**Pull request by Qix-**
Just some glaring issues in the README. :+1:

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -4,14 +4,13 @@
...
+Even though JavaScript is single-threaded, IO in Node.js can happen in parallel due to its async nature. AVA takes advantage of this and runs your tests concurrently, which is especially beneficial for IO heavy tests. In addition, test files are run in parallel as separate processes, giving you even better performance and an isolated environment for each test file. [Switching](https://github.com/sindresorhus/pageres/commit/663be15acb3dd2eb0f71b1956ef28c2cd3fdeed0) from Mocha to AVA in Pageres brought the test time down from 31 sec to 11 sec. Having tests run concurrently forces you to write atomic tests, meaning tests don't depend on global state or the state of other tests, which is a great thing!
```
Comment (Qix-) [inline readme.md]: 5 points to whoever can spot the difference.

File: readme.md [hunk 2]
```diff
@@ -111,7 +110,7 @@ Files starting with `_` are ignored. This can be useful for having helpers in th
 
 ## Documentation
 
-Tests are run async and require you to either set planned assertions `t.plan(1)`, explicitly end the test when done `t.end()`, or return a promise. [Async functions](#async-function-support) already returns a promise implicitly, so no need for you to explicitly return a promise in that case.
+Tests are run asyncronously and require you to either set planned assertions `t.plan(1)`, explicitly end the test when done `t.end()`, or return a promise. [Async functions](#async-function-support) already returns a promise implicitly, so no need for you to explicitly return a promise in that case.
```
Comment (shinnn) [inline readme.md]: sync**h**ronously

File: readme.md [hunk 3]
```diff
@@ -4,14 +4,13 @@
...
+Even though JavaScript is single-threaded, IO in Node.js can happen in parallel due to its async nature. AVA takes advantage of this and runs your tests concurrently, which is especially beneficial for IO heavy tests. In addition, test files are run in parallel as separate processes, giving you even better performance and an isolated environment for each test file. [Switching](https://github.com/sindresorhus/pageres/commit/663be15acb3dd2eb0f71b1956ef28c2cd3fdeed0) from Mocha to AVA in Pageres brought the test time down from 31 sec to 11 sec. Having tests run concurrently forces you to write atomic tests, meaning tests don't depend on global state or the state of other tests, which is a great thing!
 
 
 ## Why AVA?
 
 - Minimal and fast
 - Simple test syntax
-- Runs test files in parallel
```
Comment (sindresorhus) [inline readme.md]: Why?

File: readme.md (hunk 2 above)
Comment (Qix-) [inline readme.md]: Gah, thanks. Will fix.

File: readme.md (hunk 3 above)
Comment (Qix-) [inline readme.md]: Because the next line is pretty much the same as this. Parallel == Concurrently. It's redundant.

File: readme.md (hunk 1 above)
Comment (jamestalmage) [inline readme.md]: `tests that don't`-> `tests don't`.
Where do I redeem my points? :smile:

**Comment by sindresorhus** [comment]
Thanks @Qix- :)

**Pull request merged**